fem/bilinearform_ext.hpp

```diff
@@ -102,11 +102,16 @@ class PABilinearFormExtension : public BilinearFormExtension
 protected:
    const FiniteElementSpace *trialFes, *testFes; // Not owned
    mutable Vector localX, localY;
+   mutable Vector faceIntX, faceIntY;
+   mutable Vector faceBoundX, faceBoundY;
```

Comment:
In other places of the code "boundary" is typically abbreviated as "bdr". It can changed later maybe.